ftplugin/elixir.vim

```diff
@@ -18,3 +18,6 @@ endif
 
 setlocal comments=:#
 setlocal commentstring=#\ %s
+
+setlocal iskeyword+=.
```

Comment:
@kassio you're right, this is not good. It will change the behavior of `*` as well. I'll check if this can be solved differently.

As a last resort it could be solved by remapping `K` to a custom command. I think it would be nice to make these commands usable in elixir mode, because they are useless with Vim's default configuration anyway. Same thing applies to the `gf` command. The more I think about it, the more I embrace the idea of remapping it such that it will open a `.ex` file instead of the directory with the same name. What do you think about remapping standard commands in general?